geomesa-fs/geomesa-fs-datastore/src/main/scala/org/locationtech/geomesa/fs/FileSystemDataStoreFactory.scala

```diff
@@ -110,5 +111,6 @@ object FileSystemDataStoreFactory extends GeoMesaDataStoreInfo {
     val ConfParam         = new GeoMesaParam[Properties]("fs.config", "Values to set in the root Configuration, in Java properties format", largeText = true)
     val ReadThreadsParam  = new GeoMesaParam[Integer]("fs.read-threads", "Read Threads", default = 4)
     val WriteTimeoutParam = new GeoMesaParam[Duration]("fs.writer.partition.timeout", "Timeout for closing a partition file after write, e.g. '60 seconds'", default = Duration("60s"), systemProperty = Some(SystemPropertyDurationParam(WriterFileTimeout)))
+    val CharsetParam      = new GeoMesaParam[String]("fs.metadata.charset","Charset for transforming string to/from bytes array")
```

Comment:
we should add this to the docs under https://github.com/locationtech/geomesa/blob/master/docs/user/filesystem/usage.rst#filesystem-data-store-parameters